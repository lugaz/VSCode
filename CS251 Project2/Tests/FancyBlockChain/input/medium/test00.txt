359 array
oHlDqEoz 49.132036 4463
8b 92.190325 2593
ahldouu963 41.763864 2199
gAEi1 -72.094744 2341
OcrK -93.432720 1152
wSnIvFisU 12.808360 1615
s 167.131175 4155
0bx4FbWMYo -60.059321 4779
odJ -61.111461 3836
QIsl1FsJemhl -21.804852 1360
5K -68.039372 2497
auiG6VjVdmk2 162.328238 2082
o1j4QoIP7pp 124.686787 2006
TVK 180.531186 4395
s7HN9Bo 164.676405 4388
2UKjyLQRU 14.639728 3945
LWi8QlyyGj -91.566222 4962
07QhKHiImQ5k 64.647735 1810
EoEe0u6dFiC -53.790764 4682
jG33XIz2zh8 -121.708104 2221
Y5bk5mNKp -94.257139 2200
rNyrccEC0PfX -152.528498 2358
WB5i4Bn3NbarV -9.281490 2378
bpjT3nREK -111.153676 4628
rPkGVmrxyNb 32.680251 3022
ZXy -90.383111 3741
Xe5IVBsathmEh 31.457362 4452
w -41.974754 1230
PoQ 17.473257 3430
6N4kNa0Pyy -177.721588 3545
KckO4I6jOqEM 124.910091 3509
0UzigxVCNprMl -21.296971 1452
kC8PCmzOem 151.281994 4559
S5ZB -77.599452 3288
lCdedQ4j62jB -145.647314 4033
mMzHguB7YQ1Y -209.774468 4092
ZW6Ly 72.402341 1398
sSmAccWuqy -69.990358 1142
F 87.362627 1752
sfu1X -21.508823 2382
YKjdfdl1F -51.178918 3393
vBV 88.435499 2256
cC5h6 -11.081804 1343
UFqmykB 169.890468 2471
oV4xyW -103.425079 2824
SgSEN5zuYP 87.952105 1059
U04V7OR 124.612304 3113
odeqVdKnBnJNU -116.362557 1216
m0cet -192.917180 1066
PWeNaHQ 62.682235 2558
ZbWRZjCkdO -143.061080 1417
4cP3XpomJ 21.786636 4827
bCp -132.575672 3123
BoPjxdHJw -53.405233 4111
Bc 103.600655 4275
Z7wVaoEKznSE 154.560585 2050
rk83xPs6gaYLH -49.737645 3039
IzJzMK4LO4hi -57.895890 2974
UkrO0xc 136.756501 1741
XZsmyO 146.711449 4415
3O61 -123.098994 2635
e0D30djR -227.035247 1167
dj8iD9 -33.368627 2759
l74j -101.751114 4087
94lKiuK 71.924190 4541
pMhx -34.576665 4850
qhdlhdN6W 30.086456 1706
SpdtjI -187.460819 2610
aSQl8DcBahlf -151.152262 3890
cYcvE29MCYr -26.334478 1386
VEJ3g6G 156.161277 2713
YRJYNpo -73.195847 4209
t -26.685917 4616
WJAEbsXK -106.403232 4450
TECmU 48.257340 1304
Qz1j7lZleu1n -107.294884 3818
FSag -79.654648 4630
vV 163.305817 4548
17Zb57GuNQp -14.876354 4188
Y0imNHwnU43 -101.470132 2299
vk -39.315487 4833
WA2CSkzxxXtq 110.940875 1965
m070YwhwNQSk -96.916965 4391
VKjr1l9HHq -97.763231 1650
K 129.689750 2652
poODnPRq -87.172764 1102
RrPQFu8k 71.455056 2515
xta 73.035977 3151
PaHMyW -70.778237 4125
AsS -74.438717 4001
99wj6iR 70.480311 2037
KhfKKNfWg1l 17.366031 3970
ox25do0xA 197.759688 1244
AXJqC 155.638226 2925
y2PZa7UI 8.617126 4897
8QTwWWDTK 136.968731 3632
yewL6OgwCshBX -27.391740 3933
sYREBDjmY1x 79.272745 3550
MV 62.744671 4692
mziHu 86.350266 3211
ckHgeONS7naOF 2.790924 2327
YuE134AlMv -75.965343 4957
Up2cXs28fb -154.340573 1887
caRV -104.982058 3766
CoNuuMugisZq0 -34.594001 4351
OXYJ -11.746078 4821
eNz4uEPulK 115.905764 4926
tcsZBb -24.649193 3561
IJMniXQgU -117.251559 3671
ILDW 67.466765 1138
q8vggwAUOCB 16.004844 1269
ycN 5.863115 4480
0 -81.621190 4157
34Iva0hmy -78.209389 1118
pT0sE -10.243257 4473
BRxzCDB0fYL5 -265.151194 1033
sEsFejRfgR 73.339172 3036
PU3 -13.995202 1356
HzWlXxX9le389 94.336568 4589
B -1.442299 1060
H7boYXBKlH 80.305006 1595
1f1Ddn3 32.691135 1293
xvhjoO8aE 120.871603 1679
kF0smKq78YNKq -32.322162 2078
DctueE5CEdxDF -41.037988 3487
xbw7mS1Zje7e4 79.088845 3950
t3AAuqRRHQr 20.158863 4287
hyo5VfEYil 106.869134 1724
J4GIwpsfWWk 85.701730 2549
CxWZC -181.574982 1682
nzdsqavPlW 71.788686 3750
sXM8ysntSWw9 -22.128959 4714
S4F6s5QqU -36.572462 2707
C 56.096930 4794
9PnXnnt 104.835079 2117
ycZaJOhx 146.991995 2668
ywX5 90.214202 2799
o 56.635094 3042
QeVco 6.870166 2523
OHOJUxYASuv2 -59.410529 4251
ItdWrqDPyMKyX 105.769586 2311
cJ4NY1Rr5cH -119.243728 3734
XRpq 9.176489 4497
5dU 67.459594 1888
yRnv2Dr0 -227.031233 1083
W -7.732633 1627
Zytg8 -37.803784 1976
I9HRWW 20.468044 3073
mYk8 71.682010 2559
47HCqbWBJcv0k -49.714132 3805
udmZ9WAEo -10.731846 3401
EJQ5PfXwT 173.265116 4079
BkTLIR5wg3v 93.850560 4130
endhihnJvNZiD -31.751116 3193
yt3 -13.352303 3791
aXXDj -69.001501 3188
x 15.861410 1802
EJDO 3.677752 1455
EQ6VC02z69Ufc 99.722661 1983
JJwPiuF2Aaw -197.707705 4553
hd0wpnR -37.215735 4761
d 77.706998 2001
VPysVHVzP9Rc -25.160356 4091
ic42Wn 116.604973 4378
37ToL5yVNu 173.689579 2268
wCMxcZchf4VZ 7.803177 1667
Lejd0bYga01A 172.549945 2935
X -1.103907 2368
6uLQpJH7pmDCN 44.686870 4555
CzadX6YtnsSp -147.904150 3815
4ru4qL3K 19.059047 4995
QZWTweqIm 8.066094 3405
1Ak -76.444654 3202
LBj -146.687250 2372
Koy -117.306504 1778
auy -61.131462 3175
eSB -15.245811 3942
6 105.550445 2533
PLhGFpEe6W 151.235074 3033
ltKQ -67.567108 4588
4TqKuw7 -143.664063 3127
TR5oJ0WMy9h 21.020490 2793
soSqz3IKDZ1r -140.718673 4215
Jmn4V0l1ImptO 20.482933 3461
Y8e3eVqonLxm -45.340105 3497
3 -29.064919 2575
jPj 23.336611 1298
0yD -148.440708 1828
5eHL -54.038699 2909
jn 33.649066 4035
mEDjv6d -82.649780 4728
cZQnG3T3 -12.521244 1914
jnZQLW5i -94.934715 2345
MYXD12c -3.884414 2767
LuDo 237.239654 3209
mfyrfbwHTku -9.893695 1341
tEEq 164.831851 1895
gD65txHnAf 177.623610 2900
U4W72 48.699609 4767
3sX 12.775837 1523
oWVAWr0 -26.469205 3179
7hn3 -41.652685 3992
KnrBw9aaPEuh8 116.527853 1608
xTpJTe -4.867682 1904
g 114.128203 3512
gcA79 102.997175 2883
1t 28.733063 2773
d32K8Ndw7 -1.021051 1865
lv4Cua -30.704786 3163
PDcSHGPPOWdzg -51.136838 2435
ZBP9R7GZ -138.571169 1404
9gQZwyB4gjty -9.146118 4645
3yCNP 107.978136 3376
QUL6PyVI 125.364958 1528
EA -9.089755 2658
73khOVRdohd 9.551782 3122
ExkU -37.470950 4910
oLPP 53.412556 3385
06LSdWAY 63.892960 2373
D 36.774252 4134
xY 33.926994 1160
L -207.558914 3473
UK61rTlnMF7 -17.244799 3991
oDv3nhXl5Vld3 -54.306102 2612
Z8gTBGgQ -67.018725 4420
mXN 5.887802 1961
ZMzo0zeB 98.610223 3387
xCCRJ7A -27.901748 3018
MgZ5SCa7T8IQ 37.251076 4961
cYdzRd3 15.993003 4062
PCBvdLNC 53.742622 2539
q7RBQUFSYk0 162.577688 4231
KLfi7u8Rf9z -34.974790 3064
p6dpvNtQ -25.328135 4650
n5xC -0.127832 1457
UR0m 71.771508 2162
G 88.092946 3655
nhhr 207.639354 2461
BO -56.694491 1934
3AOQt8pi -114.217755 1407
Zrr5p70ahRZcX 57.459686 3149
74QvHigN6QU 38.364830 1756
6DMuLqeM 1.487557 3166
5o -112.826760 2920
4 -7.204106 4299
8tQA3tk21l3Ps 20.531702 1990
plj0oD -224.586662 3911
7QE7xQ2aK 118.501433 4419
vzdxVpuWEbF -57.777980 4740
xHee4810W -163.284319 2446
j4dC8f 129.439390 2981
xSYqpjqHUPx 16.106370 1855
z -66.095166 1116
FP4lFYvI1UAmr 54.957046 3244
LzCJWfan1xMf 77.193333 4093
clMFMFy 69.192140 3494
PoB -111.000888 2532
Fx6V 133.205138 3455
CpbHWlP3lD 12.880169 1136
lKo -117.816485 2724
8hdzPHcSnui 125.691708 4583
6LTm5RVjKye 5.606709 3351
WjCR0q5UExIT 165.527664 4479
Jp -137.874122 3034
TclYwqJ1iG8ZV 205.087297 1358
7u0eHFj0ppnV -102.363600 3611
woM -24.781718 4077
ZNSvhTuUM0iHr 68.393945 2333
jCwXr7z6 -166.305128 4871
0cDVGqBN 163.612991 1031
ucQW5OY0x -50.046902 4747
t8YoVdVPd -93.562494 3764
HQmP8qn6gxFK -23.660664 2727
Xg5 -149.020635 2554
Jbmm1wz -202.384452 2132
hnp3Nl -17.114197 2664
YgDpo3L7awbxc 42.633694 3365
lfEYmIs -62.014184 1645
cqQDbM9o5Xk8G 68.344805 4557
iA 22.266675 4237
8wKlXYO -37.756285 2101
FCvNfW9 96.306331 2905
gMXKNydwg4nC2 -52.494256 2490
TA -146.796782 4684
BQmRd3Lz 75.630288 1308
jow2PdVxv4tn8 -42.915602 4099
cvCmt43 197.697503 2352
aEdBdx3J38 -90.646411 3682
sBs5BHRrcyOn -3.355266 4620
OcUn2ggU 99.777462 3759
c2u 32.336173 1381
VntU -146.319104 1954
AKE 128.709838 2495
d9QX9vPheY 115.661972 2762
NLWa -45.808037 1036
Y05SxJw -59.948959 3352
RW3ciBlU -28.462766 4861
hrX0TAjamK6AB -117.070196 1536
tW2vBkXI 78.360435 4973
SqSNhjdcC -24.323999 1521
eSH1J34qsNSJl -46.123646 2678
Ep7 -14.344008 2322
kaTtIqhdKUlH -88.444285 3370
bwYPIe -63.353607 3468
MNLLNa -107.868335 2265
2wHjG 42.040338 1631
b 33.561755 3592
H4kB 19.640513 3406
y 66.034426 3760
XDa5AGffal1 127.299115 3716
Z1 -16.768028 4518
uGys5kTpa -183.663497 4269
CaO31mXo 113.881656 4545
GRjL 154.231217 1032
cfMsnetg1uFec -45.911544 4211
M5aJ 26.205869 1493
YB 139.095719 3465
3alPjIJ46h6s -94.818655 4213
yjktUzki 70.414354 3237
suHRXzWN -26.804593 3547
NErHsP4DWXtb -258.149000 2011
aZYwr2YkzAAEF -40.003119 2121
dIYppCpdhD 110.029512 1443
L1ABJPQwr -112.638678 2066
Qd7 29.889456 3068
wIBvyPXza 15.122906 2507
KFdi 41.638222 2684
P 39.836192 4288
Lv2tbT9z -79.884710 3292
Z3fAPYH7yQVe -53.949979 3040
UmvefdJHT 6.876058 2556
cVsueR 48.418989 2795
9 -287.770800 1043
TqZfhO7 51.905622 2396
alrJM2 135.175646 3879
ZcHHd6JvkKnbq -109.267330 1832
MAA0Uz3 19.040503 3375
AnNkYFlfm -30.640865 3807
3KoGbP -166.846111 3696
J4no -14.974679 3821
63 71.786867 3893
C0Q 82.398185 3046
LGXk8EoJGDf 18.207968 4820
Vlx0vDme -168.811764 3549
LDgcR6McXKx 39.469238 3691
hsiv 168.255014 1233
xpkYhfvmfRj 15.827261 2567
xk15P0hpTEW -158.793264 4818
D22k 147.225390 4470
E -66.870030 3902
Oh -32.443964 2456
eEgHQwRAA -0.350473 1694
u1wcxGvC 118.495159 4055
c0kkigktORw9 93.670217 1416
pQO5o0MwbLv90 65.627897 4184
2W2L 24.048846 1024
jWefxKPZHw 117.968205 3159
9ED0YwXO 81.997340 4977
fZDOV -185.951253 1332
getBlock ZMzo0zeB
getBlock Vlx0vDme
getEarliestBlock
getEarliestBlock
removeBlock TA
updateEarliestBlock 122.449317
getBlock ZXy
getBlock 7Oy
updateBlock NLWa 2.022130
addBlock Zap30F 20.319869 703
getEarliestBlock
addBlock 070Qj6laGmG 13.705826 2572
removeBlock ZMzo0zeB
getEarliestBlock
getBlock q
getEarliestBlock
getEarliestBlock
removeBlock XDa5AGffal1
updateBlock hd0wpnR -44.617634
updateEarliestBlock -202.100734
removeEarliestBlock
addBlock LWSQr -65.117582 4624
getBlock RW3ciBlU
addBlock v 170.970086 1210
removeBlock htj5d
addBlock 94KaTPAT -155.552579 3589
removeBlock 7u0eHFj0ppnV
addBlock rDi1E1lB -217.360758 4028
getEarliestBlock
updateBlock jWefxKPZHw 203.776020
addBlock Z 22.664365 2104
addBlock 55E3at5 13.744831 505
removeBlock BRxzCDB0fYL5
getEarliestBlock
addBlock raivQX2WxGIP -145.051648 4993
removeBlock mXN
addBlock WmCrQO -31.930132 571
addBlock WnC1mHefAABN 85.722377 2234
updateEarliestBlock -115.936498
updateBlock 3O61 -127.996386
addBlock XDa5AGffal1 154.094856 1524
removeEarliestBlock
getEarliestBlock
updateBlock ltKQ -104.216467
addBlock xtwfan 1.013372 2858
getEarliestBlock
removeEarliestBlock
getBlock zyXImpBBZiwr
removeBlock h8c9ST3Py
removeBlock 94lKiuK
removeEarliestBlock
addBlock fMDpt4U0IAJ 32.671653 4441
removeEarliestBlock
removeBlock dj8iD9
removeEarliestBlock
removeEarliestBlock
getBlock 3sX
updateEarliestBlock -33.275125
addBlock Wkw0itu -42.855781 686
updateBlock bwYPIe 59.658587
addBlock SqG -14.399362 2431
addBlock I 51.992732 1826
updateBlock F -158.253586
getEarliestBlock
removeEarliestBlock
removeEarliestBlock
addBlock fshn -18.444174 4865
addBlock 49qUx 202.531183 4940
updateEarliestBlock -140.633590
removeEarliestBlock
removeEarliestBlock
updateBlock tW2vBkXI 43.658793
addBlock QWS6 49.169784 3562
getEarliestBlock
updateBlock caRV 63.567281
getBlock PDcSHGPPOWdzg
addBlock 54Q 149.969315 4523
removeEarliestBlock
addBlock CXt7JpZGWby -69.601165 1549
getBlock YKjdfdl1F
updateEarliestBlock -210.979245
updateBlock clMFMFy 7.143063
addBlock it -113.276785 68
getBlock 0UzigxVCNprMl
updateEarliestBlock 30.427174
removeBlock UFqmykB
removeBlock 0bx4FbWMYo
addBlock eaB 48.571337 1786
updateEarliestBlock -14.717707
addBlock 5dIrueo -101.367978 62
addBlock ig17w9Om 150.665396 1184
getEarliestBlock
addBlock V 18.386242 599
removeBlock P
addBlock 6Iukz9Lnt 11.928791 661
updateBlock HQmP8qn6gxFK 62.245770
removeBlock odeqVdKnBnJNU
getBlock TqZfhO7
addBlock lk 138.672096 3172
getBlock VukNXiq25IPUQ
removeBlock ig17w9Om
updateEarliestBlock -13.851008
getEarliestBlock
updateBlock clMFMFy 55.710824
getEarliestBlock
removeBlock UK61rTlnMF7
getEarliestBlock
addBlock OSBb -67.079916 2629
updateBlock VKjr1l9HHq 22.179788
addBlock IaDI2BxMXj6ld 58.554639 4859
addBlock ULPZ -40.676675 2801
removeBlock aEdBdx3J38
addBlock j298bqoKV 27.580996 1264
removeEarliestBlock
addBlock 6ewAwVFy -3.316465 3456
removeEarliestBlock
getEarliestBlock
removeEarliestBlock
removeBlock LWSQr
getBlock U4W72
getEarliestBlock
getBlock mfyrfbwHTku
getEarliestBlock
updateBlock Z4 -45.439044
updateEarliestBlock 35.534859
addBlock utNDgal2WT 117.129311 2832
updateBlock ZNSvhTuUM0iHr -90.820090
addBlock rXkhmyJBtnes3 2.879449 4756
getBlock o
addBlock NUd4qu -23.011247 1875
getBlock EA
updateEarliestBlock -50.723711
updateBlock Z3fAPYH7yQVe -88.127588
updateEarliestBlock 33.831015
updateBlock dIYppCpdhD -185.254991
addBlock gZDi -7.791364 2021
addBlock ay6kigJOnO 58.417406 1260
updateEarliestBlock -8.469291
updateBlock hd0wpnR -19.653567
updateEarliestBlock 54.953218
addBlock zTwsf3o -104.246203 1567
getEarliestBlock
addBlock qrsJt 138.026468 2466
addBlock SFu 151.190838 425
updateEarliestBlock 93.129576
getEarliestBlock
updateEarliestBlock -27.695915
updateBlock endhihnJvNZiD -60.732089
getBlock 7QE7xQ2aK
removeBlock 8QTwWWDTK
getEarliestBlock
getBlock AXJqC
updateBlock rNyrccEC0PfX -81.599229
getEarliestBlock
updateBlock vk 123.258263
getEarliestBlock
removeBlock cZQnG3T3
removeEarliestBlock
removeBlock ULPZ
addBlock M4c9wkLq1 102.877745 1672
updateBlock KFdi -129.612186
updateEarliestBlock 94.518892
updateBlock sXM8ysntSWw9 -94.375158
updateBlock IaDI2BxMXj6ld 29.916407
removeEarliestBlock
getEarliestBlock
getEarliestBlock
updateEarliestBlock 159.531826
getEarliestBlock
removeBlock QWS6
updateEarliestBlock 80.457388
addBlock A8s1agiLsg 22.705639 4398
updateBlock NErHsP4DWXtb -69.997097
addBlock YMEFd6n3mVd3w 228.444725 1655
getEarliestBlock
updateEarliestBlock -165.675443
removeEarliestBlock
getEarliestBlock
getBlock xY
removeEarliestBlock
updateEarliestBlock -4.671506
addBlock mPpTd8fc -41.091811 3849
addBlock cZQnG3T3 -60.270240 3837
getBlock kF0smKq78YNKq
addBlock mZ 103.907930 2660
updateEarliestBlock -6.422901
getEarliestBlock
addBlock Au45JPyivQ -119.743823 3965
removeBlock 3alPjIJ46h6s
updateEarliestBlock -11.794530
addBlock ULPZ 63.344087 3742
removeBlock 74QvHigN6QU
updateEarliestBlock -47.269434
removeEarliestBlock
addBlock TOuTLwSzy4Dte -100.124013 651
getBlock 5dU
removeBlock ZbWRZjCkdO
getEarliestBlock
