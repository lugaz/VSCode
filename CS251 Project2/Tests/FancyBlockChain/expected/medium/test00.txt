ZMzo0zeB 98.610223 3387
Vlx0vDme -168.811764 3549
2W2L 24.048846 1024
2W2L 24.048846 1024
TA -146.796782 4684
ZXy -90.383111 3741
null
true
Zap30F 20.319869 703
true
ZMzo0zeB 98.610223 3387
0cDVGqBN 163.612991 1031
null
0cDVGqBN 163.612991 1031
0cDVGqBN 163.612991 1031
XDa5AGffal1 127.299115 3716
GRjL 154.231217 1032
true
RW3ciBlU -28.462766 4861
true
null
true
7u0eHFj0ppnV -102.3636 3611
true
BRxzCDB0fYL5 -265.151194 1033
true
false
null
9 -287.7708 1043
true
mXN 5.887802 1961
true
true
true
m0cet -192.91718 1066
yRnv2Dr0 -227.031233 1083
true
yRnv2Dr0 -227.031233 1083
yRnv2Dr0 -227.031233 1083
null
null
94lKiuK 71.92419 4541
poODnPRq -87.172764 1102
true
z -66.095166 1116
dj8iD9 -33.368627 2759
34Iva0hmy -78.209389 1118
CpbHWlP3lD 12.880169 1136
3sX 12.775837 1523
true
true
true
Wkw0itu -42.855781 686
Wkw0itu -42.855781 686
sSmAccWuqy -69.990358 1142
true
true
xY 33.926994 1160
e0D30djR -227.035247 1167
true
v 170.970086 1210
PDcSHGPPOWdzg -51.136838 2435
true
v 170.970086 1210
true
YKjdfdl1F -51.178918 3393
true
0UzigxVCNprMl -21.296971 1452
UFqmykB 169.890468 2471
0bx4FbWMYo -60.059321 4779
true
true
true
5dIrueo -101.367978 62
true
P 39.836192 4288
true
odeqVdKnBnJNU -210.979245 5010
TqZfhO7 51.905622 2396
true
null
ig17w9Om 150.665396 1184
V 18.386242 599
V 18.386242 599
UK61rTlnMF7 -17.244799 3991
V 18.386242 599
true
true
true
aEdBdx3J38 -90.646411 3682
true
6Iukz9Lnt 11.928791 661
true
hsiv 168.255014 1233
ox25do0xA 197.759688 1244
ox25do0xA 197.759688 1244
LWSQr -65.117582 4624
U4W72 48.699609 4767
j298bqoKV 27.580996 1264
mfyrfbwHTku -9.893695 1341
j298bqoKV 27.580996 1264
true
true
o 56.635094 3042
true
EA -9.089755 2658
true
false
true
mfyrfbwHTku -9.893695 1341
true
false
PU3 -13.995202 1356
7QE7xQ2aK 118.501433 4419
8QTwWWDTK 136.968731 3632
TclYwqJ1iG8ZV 205.087297 1358
AXJqC 155.638226 2925
TclYwqJ1iG8ZV 205.087297 1358
TclYwqJ1iG8ZV 205.087297 1358
cZQnG3T3 -12.521244 1914
TclYwqJ1iG8ZV 205.087297 1358
ULPZ -40.676675 2801
true
c2u 32.336173 1381
cYcvE29MCYr -26.334478 1386
cYcvE29MCYr -26.334478 1386
ZW6Ly 72.402341 1398
QWS6 49.169784 3562
true
true
ZBP9R7GZ -138.571169 1404
3AOQt8pi -114.217755 1407
c0kkigktORw9 93.670217 1416
null
c0kkigktORw9 93.670217 1416
true
true
kF0smKq78YNKq -32.322162 2078
true
EJDO 3.677752 1455
true
3alPjIJ46h6s -94.818655 4213
true
74QvHigN6QU 38.36483 1756
M5aJ 26.205869 1493
true
5dU 67.459594 1888
ZbWRZjCkdO -4.671506 5040
TOuTLwSzy4Dte -100.124013 651
